### PR TITLE
Adding the environment magento-cloud to be used by den env-ini command.

### PR DESCRIPTION
**Pull request by henriquekieckbusch**
It adds the environment "magento-cloud"
So the command `den env-init <name> magento-cloud` will create a magento2's .env file, but all the services versions come from your Magento Cloud project.

### CODE REVIEW

**Review by navarr — COMMENTED** [review]
It looks good!  Are you willing to update it to detect Elasticsearch vs OpenSearch?

**Comment by henriquekieckbusch** [comment]
Hey @navarr ! Sure, it is possible, let me update it soon.

**Review by navarr — APPROVED** [review]

**Review by navarr — CHANGES_REQUESTED** [review]
This needs to be able to detect whether or not the folder we're initializing in is a Magento Cloud project.  Right now, it does not do that successfully.  It does prompt for the Magento Cloud project and environment, but it's not entirely clear (to me) why it's doing that.

Additionally, when told to import environment variables, It generated this file:

```yaml
version: '3.5'
services:
  php-fpm:
    environment:
```

And displayed the error that "Variable not found: env:COMPOSER_AUTH".  `env:COMPOSER_AUTH` is configured on the Project level.

**Comment by henriquekieckbusch** [comment]
> This needs to be able to detect whether or not the folder we're initializing in is a Magento Cloud project. Right now, it does not do that successfully. It does prompt for the Magento Cloud project and environment, but it's not entirely clear (to me) why it's doing that.
> 
> Additionally, when told to import environment variables, It generated this file:
> 
> ```yaml
> version: '3.5'
> services:
>   php-fpm:
>     environment:
> ```
> 
> And displayed the error that "Variable not found: env:COMPOSER_AUTH". `env:COMPOSER_AUTH` is configured on the Project level.

it should be right on this last commit. I'm working on a new update (to create the env.php file) but I think I will wait sometime to finish that, so maybe a new PR later.

File: environments/magento-cloud/init.env.cmd [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        then
+            VAR=$(magento-cloud vget --property=value --level=environment --project=$PROJECT --environment=$ENVIRONMENT $LINE)
+            LINE=$(echo $LINE | sed 's/env://g')
+            echo "      $LINE: '$VAR'" >> .warden/warden-env.yml
+        fi
+    done
+}
+
+while true; do
+    read -p $'\033[32mDo you want to import Environment variables? y/n\033[0m ' resp
```
Comment (navarr) [inline environments/magento-cloud/init.env.cmd:107]: Can we include the `$ENVIRONMENT` variable in here?  I'd make a suggestion but I've never used read so I'm not entirely clear on the format to use.

But with text something like "Do you want to import the Environment variables from $ENVIRONMENT?"

**Comment by navarr** [comment]
It's working really well!  I have a single suggestion to make some phrasing more explicit

**Review by navarr — APPROVED** [review]

**Pull request merged**